### ISSUE TITLE
Remove `l-side-by-side`

### ISSUE THREAD

**Issue by DaveOrDead** [campaignmonitor/shell#63]
This layout module doesn't seem to give us anything we can't achieve with a 2 column grid.

Investigate and remove it if above concern is valid

**Comment by chris-pearce** [comment]
I think it's valid. @kllevin FYI.

**Comment by chris-pearce** [comment]
What about _reverse_ functionality side-by-side provides:

``` scss
/**
 * Modifier: reversed.
 */

.l-side-by-side--reversed {
    flex-direction: row-reverse;
}
```

Not worry about it?

**Comment by DaveOrDead** [comment]
I'm torn a bit
1. Maybe we should ditch it, because messing around with the source order from the CSS can be an accessibility nightmare.  I'm sure we can figure out a smart way around it.
2. If we think it's valid, we could add it to the grid and it will still work right?

**Comment by chris-pearce** [comment]
LGTM

**Comment by chris-pearce** [comment]
Fixed in #67.
